docker/env/version

```diff
@@ -1 +1 @@
-0.29
+0.30
```

Comment:
we have this version in a different PR: https://github.com/scylladb/scylla-cluster-tests/pull/797
make sure to update this value before merging this PR

Reply:
Ok. Thanks